core/src/main/scala/kafka/server/metadata/ZkMetadataCache.scala

```diff
@@ -60,7 +61,8 @@ trait ZkFinalizedFeatureCache {
  *  A cache for the state (e.g., current leader) of each partition. This cache is updated through
  *  UpdateMetadataRequest from the controller. Every broker maintains the same cache, asynchronously.
  */
-class ZkMetadataCache(brokerId: Int, metadataVersion: MetadataVersion, brokerFeatures: BrokerFeatures)
+class ZkMetadataCache(brokerId: Int, metadataVersion: MetadataVersion,
```

Comment:
can we do an
```
abc(
a
b
c
) {
}
```
thing here? Sorry, these long scala declarations are ugly otherwise...